modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/slot/Slot.java

```diff
@@ -18,19 +18,28 @@
 
 package org.wso2.andes.kernel.slot;
```

Comment:
Can't we have unit tests for this class? IMO some change here will lead to subtle changes in functionality which might be really difficult to debug.